src/bidiMapper/domains/network/NetworkRequest.ts

```diff
@@ -230,7 +230,14 @@ export class NetworkRequest {
   onRequestPaused(
     params: Protocol.Fetch.RequestPausedEvent,
     networkStorage: NetworkStorage
-  ) {
+  ): void {
+    if (this.#isIgnoredEvent()) {
```

Comment:
It's `continueRequest`, so maybe we should do this only for `phase == Network.InterceptPhase.BeforeRequestSent`?

Reply:
From CDP docs `Fetch.requestPaused` - `Issued when the domain is enabled and the request URL matches the specified filter. The request is paused until the client responds with one of continueRequest, failRequest or fulfillRequest. ` so I don't know when `continueRequest` is needed.

Reply:
OK, let's try. This is moving towards the right direction either way.